docs/DeveloperGuide.md

```diff
@@ -4,26 +4,162 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
-
+## Design
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Good Class
+#### Description
+The goods class keeps track of the various inventory that will be input into the system through the commands class.
+The diagram below shows the model component of the orderline class.
+![Good Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/AY2122S2-CS2113T-T09-4/tp/master/docs/diagrams/Good.puml)
```

Comment:
I think the UML format does not really follow the standards we learnt in lecture. Eg we use "+", "-", "#" or "~" to indicate the visibilities, but in the UML diagram shown it used a red square and green circle

Also the return type should be at the end instead. Eg:
getId(): Int
instead of:
int getId()